### DISCUSSION TITLE
Prompt Admin

### DISCUSSION THREAD

**Discussion by ervet** [danny-avila/LibreChat#4399] [General]
Hello,

just a quick question. How can you can control how is the "admin" user in the prompt interface. That means who gets the Manage button at the top right corner.

That is not clear. We have 2 instances for testing and the button is one visible and on the other not.

Greetings
Erik

**Comment by danny-avila** [comment]
Hi, the first created user is considered the admin. More RBAC settings will eventually come to the app but you can currently edit the database user documents to change the role

**Reply by ervet** [reply]
Hi,

I was thinking the same. Is their any documentation regarding user mgmt database ? Connecting etc. 

Greetings
Erik

**Reply by ervet** [reply]
Reply to myself. I have found everything.

Deploy mongodb with ports open and remove the --noauth switch if applicable

The rest ist straightforward:

wget https://downloads.mongodb.com/compass/mongodb-mongosh_2.3.2_amd64.deb
dpkg -i mongodb-mongosh_2.3.2_amd64.deb
mongosh mongodb://127.0.0.1:27018/LibreChat

db.users.find({ role: 'ADMIN' }).pretty()

// find the Object ID if available 
// if nothing is available chose one 

db.users.updateOne(
  { _id: ObjectId('SUPERID') },
  { $set: { role: 'ADMIN' } }
)
